src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-cluster-tabs/smb-cluster-tabs.component.html

```diff
@@ -0,0 +1,21 @@
+<ng-container *ngIf="selection">
+  <nav
+    ngbNav
+    #nav="ngbNav"
+    class="nav-tabs"
```

Comment:
shouldn't we move to [carbon-tabs?](https://carbondesignsystem.com/components/tabs/code/)

Reply:
Yes, I have updated the PR using carbon tags now. Also updated the video in PR description